### PR TITLE
elminate internal json tags forever

### PR DESCRIPTION
**Pull request by deads2k**
Removes json tags on internal `runtime.Objects` and adds a unit test to be sure that they stay that way.

@liggitt

### CODE REVIEW

File: pkg/api/latest/tags_test.go [hunk 1]
```diff
@@ -21,10 +21,6 @@ func TestDescriptions(t *testing.T) {
 		seen := map[reflect.Type]bool{}
 
 		for _, apiType := range kapi.Scheme.KnownTypes(version) {
-			if !strings.Contains(apiType.PkgPath(), "openshift/origin") {
```
Comment (liggitt) [inline pkg/api/latest/tags_test.go]: this requires descriptions on all k8s objects now as well?

Comment (liggitt) [inline pkg/api/latest/tags_test.go]: oh, nm.. just moved

Comment (deads2k) [inline pkg/api/latest/tags_test.go]: > oh, nm.. just moved

Yeah, I saw the transitive error in my ways.

**Comment by liggitt** [comment]
LGTM

**Comment by deads2k** [comment]
[merge]

**Pull request merged**